app/views/opportunity_matches/_activate_table.haml

```diff
@@ -30,6 +33,10 @@
                 = link_to_if client.accessible_by_user?(current_user), name, client_path(client)
 
               %td.text-center= priority_value(client)
+              - if supporting_data_columns.present?
+                - supporting_data_columns.map do |title, method|
+                  - value = client.send(method)
+                  %td.text-center= [TrueClass, FalseClass].include?(value.class) ? checkmark(value) : value
```

Comment:
![Screen Shot 2022-08-24 at 4 51 37 PM](https://user-images.githubusercontent.com/5333982/186521152-5ff137d0-598b-4652-8b40-d6cd7e71ca50.png)

Reply:
The existing match routes use a method `client_prioritization_value_method` Which names a method to be called on `Client` -- I like this better, maybe extend this new API into the places where we have multiple values? (The TC schemes: `DaysHomelessVeteransFamiliesYouth` and `DaysHomelessPopulations`, and the ones with tie breakers)

Reply:
Updated DaysHomelessVeteransFamiliesYouth and DaysHomelessPopulations